docs/static/Devel/LocalGameTutorial.md

```diff
@@ -1,828 +0,0 @@
-<!-- @format -->
-
-# Local Game tutorial :lion:
```

Comment:
Again  that's quite some bit of documentation that is flushed out. There is really nothing to save/adapt/maintain ?

Reply:
DITTO it would be more simple to make a new one